### PR TITLE
Add Write The Docs EU

### PR DESCRIPTION
**Pull request by bexelbie**
Dates announced here:

http://www.writethedocs.org/conf/eu/2017/news/recap-retrospective/

Site not up

### CODE REVIEW

File: data/events/writethedocs-eu-2018.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+    "accessibility": "",
+    "code_of_conduct": "http://www.writethedocs.org/code-of-conduct/",
+    "event_end": "2018-09-13",
+    "event_start": "2018-09-11",
```
Comment (rollandf) [inline data/events/writethedocs-eu-2018.json]: According to http://www.writethedocs.org/conf/eu/2017/news/recap-retrospective/
the dates are:
Prague conference, 9-11 September 2018

**Comment by szabgab** [comment]
thanks. posted here: https://codeandtalk.com/e/writethedocs-eu-2018

Will you remember updating the JSON file when the website comes live?
In case you are not one of the conf organizers, would you let them know about our repo and ask them to update it when they have the site?

File: data/events/writethedocs-eu-2018.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    ],
+    "location": {
+        "city": "Prague",
+        "country": "Czech Republic",
+        "state": ""
+    },
+    "name": "Write The Docs EU",
+    "tags": [],
+    "twitter": "writethedocs",
+    "videos_url": "http://pyvideo.org/",
```
Comment (rollandf) [inline data/events/writethedocs-eu-2018.json]: Leave this entry empty. pyvideo.org is for pycon/python conferences.

File: data/events/writethedocs-eu-2018.json (hunk 1 above)
Comment (bexelbie) [reply data/events/writethedocs-eu-2018.json]: Fantastic catch - I thnk the reversal of the entries caused by brain to misfire.  As this PR merged, I'll send a fix up.

File: data/events/writethedocs-eu-2018.json (hunk 2 above)
Comment (bexelbie) [reply data/events/writethedocs-eu-2018.json]: I read over this line - another great catch!

**Comment by bexelbie** [comment]
no fixup needed -fixed already in de071f2e21e239a1ce521b49c9aa7f4ec7069491

**Comment by szabgab** [comment]
Hi, if you sent this in PR because of the Hacktoberfest, there are 3 more days left. Please, tell you friends there are plenty more PR opportunities here. In 3 days they can get a cool T-shirt as well. Please tweet about it mentioning the web site https://codeandtalk.com/ and referencing me @szabgab so I can easily reshare. - Thanks

**Pull request merged**